docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -7,6 +7,14 @@
 
 The ServiceNow connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow incidents.
 
+[float]
+[[servicenow-connector-prerequisites]]
+==== Prerequisites
+Create an integration user in ServiceNow and assign it the following roles.
```

Comment:
I'm not all that familiar with ServiceNow, but is "integration user" correct?  Does the user need these roles to use the Kibana UI?

Reply:
Changing to "ServiceNow integration user" as per our Zoom convo.